projects/kernel-config/Makefile

```diff
@@ -12,7 +12,7 @@
 HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
 # Name on Hub
 IMAGE:=kernel
-ARCH?=$(shell if [ "$$(arch)" = "x86_64" ]]; then echo x86; else $$(arch); fi)
+ARCH?=x86
```

Comment:
I may have asked this before, but what's the rational for `x86`? I would prefer using the kernel arch for this